libvirt/tests/src/passthrough/pci/libvirt_pci_passthrough_eeh.py

```
@@ -0,0 +1,401 @@
+            try:
+                idx = logs.index(last_dmesg_line)
+                new_logs = logs[idx + 1:]
+            except ValueError:
+                new_logs = logs
+        else:
+            new_logs = logs
+        return new_logs, logs[-1] if logs else None
+
+    def test_eeh_nic():
+        cmd = f"echo {max_freeze} > /sys/kernel/debug/powerpc/eeh_max_freezes"
+        process.run(cmd, shell=True)
```

Ruff (0.13.1)

Function call with `shell=True` parameter identified, security issue

(S604)